collect_app/src/main/java/org/odk/collect/android/application/Collect.java

```diff
@@ -276,12 +276,17 @@ public void onCreate() {
 
         if (BuildConfig.BUILD_TYPE.equals("odkCollectRelease")) {
             Timber.plant(new CrashReportingTree());
-        } else {
+        } else if (BuildConfig.BUILD_TYPE.equals("debug")) {
             Timber.plant(new Timber.DebugTree());
+
+            // Robolectric tests fail in strict mode + leak canary, so disable them
+            if (!"robolectric".equals(Build.FINGERPRINT)) {
```

Comment:
Huh I didn't know you could use `FINGERPRINT` for this. I think I'd actually prefer we use the `TestCollect` class that Robolectric automatically references for customizing the test application. This keeps test code out of our `Collect` class.